src/screens/Settings/components/DeleteAccountDialog.tsx

```diff
@@ -0,0 +1,292 @@
+              </Trans>
+            )}
+          </Text>
+        </View>
+
+        {stage === Stages.SendEmail && (
+          <Admonition>
+            <Trans>
+              You can also{' '}
+              <RNText
+                style={[{color: t.palette.primary_500}, web(a.underline)]}
+                onPress={() =>
+                  control.close(() => deactivateDialogControl.open())
+                }>
+                temporarily deactivate
+              </RNText>{' '}
+              your account instead, and reactivate it at any time.
+            </Trans>
+          </Admonition>
+        )}
+
+        {stage === Stages.DeleteAccount && (
+          <View style={[a.gap_md]}>
+            <View>
+              <TextField.LabelText>
+                <Trans>Confirmation Code</Trans>
+              </TextField.LabelText>
+              <TextField.Root>
+                <TextField.Input
+                  label={_(msg`Confirmation code`)}
+                  placeholder={_(msg`Enter code from email`)}
+                  value={confirmCode}
+                  onChangeText={setConfirmCode}
+                  autoCapitalize="none"
+                  autoCorrect={false}
+                  autoComplete="one-time-code"
+                />
+              </TextField.Root>
+            </View>
+            <View>
+              <TextField.LabelText>
+                <Trans>Password</Trans>
+              </TextField.LabelText>
+              <TextField.Root>
+                <TextField.Input
+                  label={_(msg`Password`)}
+                  placeholder={_(msg`Enter your password`)}
+                  value={password}
+                  onChangeText={setPassword}
+                  secureTextEntry
+                  autoCapitalize="none"
+                  autoComplete="password"
+                />
+              </TextField.Root>
+            </View>
+            <View>
+              <TextField.LabelText>
+                <Trans>Handle of the account you want to delete</Trans>
+              </TextField.LabelText>
+              <TextField.Root>
+                <TextField.Input
+                  label={_(msg`Your handle`)}
+                  placeholder={currentAccount?.handle}
+                  value={handleInput}
+                  onChangeText={setHandleInput}
+                  autoCapitalize="none"
+                  autoCorrect={false}
+                />
+              </TextField.Root>
+            </View>
+          </View>
+        )}
+
+        <View style={[a.gap_sm, gtMobile && [a.flex_row_reverse, a.ml_auto]]}>
+          {stage === Stages.SendEmail ? (
+            <>
+              <Button
+                label={_(msg`Send Email`)}
+                variant="solid"
+                color="negative"
+                size="large"
+                disabled={isProcessing}
+                onPress={onPressSendEmail}>
+                <ButtonText>
+                  <Trans>Send Email</Trans>
```

Comment:
```suggestion
                  <Trans>Send email</Trans>
```
All but one use of this string (another use in the `DeleteAccount` modal) use sentence case, would that work here for this button?

![IMG_0960](https://github.com/user-attachments/assets/3c54c833-9afc-49c2-9505-ea58c7e5a387)